parser-typechecker/src/Unison/Codebase/SqliteCodebase/Operations.hs

```diff
@@ -731,3 +734,15 @@ makeMaybeCachedTransaction size action = do
   pure \x -> do
     conn <- Sqlite.unsafeGetConnection
     Sqlite.unsafeIO (Cache.applyDefined cache (\x -> Sqlite.unsafeUnTransaction (action x) conn) x)
+
+insertProjectAndBranch :: Db.ProjectId -> ProjectName -> Db.ProjectBranchId -> ProjectBranchName -> Sqlite.Transaction ()
```

Comment:
Moved this down a level from `ProjectCreate.hs` so I can use it in more places :)